models.yml

```diff
@@ -34,6 +437,106 @@ mrc-ide-covid-sim:
   supportedParameters:
     - interventionStrategies
     - r0
+  supportedRegions:
+    - AT: []
```

Comment:
I like this better too. Empty array of subregions is more meaningful than `null`/`undefined.`

Reply:
See my comment above. Here, I think we should be showing the ‘_self’ or ‘all’ region only.